### DISCUSSION TITLE
2024.03.08 회의

### DISCUSSION THREAD

**Discussion by yusoonsun** [bitcamp-teams/mos#14] [회의록]
**참석자 : 김준회, 최성원, 박준수, 권채린, 김선종, 유순선**

# 주제: 스터디 커뮤니티 사이트(모두의 스터디; 모스)

---

## 🧠 UI 프로토타입 완성하기

- 알림창 구현 필요
- 회원 정보 수정 페이지 (깃, 스택오버플로우 연동설정, 연동해제, 탈퇴)
    - 프로필 아이콘 누르면 뜨는 서브 메뉴 ( 마이페이지, 스터디, 회원 정보 수정, 로그아웃)
- 로그인 후 회원아이콘 클릭하면 홈, 마이페이지, 스터디, 회원정보수정 목록
- 스터디 모집 글 작성
- 스터디 참여 회원 목록에 불량 회원 신고 기능
- 스터디 페이지
    - 즐겨찾기 한 스터디 목록
    - 전체 스터디 목록 ( 버튼 놓고 숨길수 있게)
        - 스터디 한 줄 설명
    - 스터디 별 기술 스택 이미지
- 스터디 산출물 페이지
    - 리스트 형식
- 마이페이지
    - 작성중인 글 목록, 내가 쓴 글에 달린 댓글
    - 좋아요 목록
    - 팔로우 , 팔로잉 (깃허브참고)
- 내가 쓴 글 & 댓글 페이지

### 👀 기능 고민

- 스터디 관리 페이지에 투두리스트 기능이 있으면 어떨까?
- 작성중인 글 자동 저장 기능 필요한지?
- 스터디 모집 인원은 스터디 개설한 사람이 변경가능
- 완료한 스터디 목록에 스터디 기간표시
- 스터디 참여 회원 목록에 불량 회원 신고 기능
- 참여 스터디 숨김 기능 (숨겨진 스터디 목록 화면)
- 회원 팔로우 기능 ( 글작성하면 알림기능)

### 🧏🏻‍♀️ 주말 과제

- 채린 : 회원정보 수정 페이지, 프로필 누르면 뜨는 서브메뉴
- 준수 : 모집화면 , 모집 글 작성 페이지, 알림창
- 준회: 스터디 계획 입력 페이지, 스터디 자료 게시 페이지
- 성원 & 선종: 스터디 현황 페이지
- 순선: 메인페이지 &  스터디 검색페이지 , 마이페이지

[Uploading 2024.03.08_회의.pdf…]()

**Comment by joonhoekim** [comment]
[발표자료 - 모두의 스터디.pdf](https://github.com/bitcamp-teams/mos/files/14832831/-.pdf)
최대한 간단하게 만들었던, 프로젝트 주제선정 발표자료입니다.
